components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -394,6 +394,13 @@ Rules of type `Node` can have conditions with attributes `Chef Organization`, `C
 Navigate to the project list page once more. Select `Update Projects`. Upon completion of the update, you should be able to
 filter by `project-devops` across Automate's dashboards and see only the ingested data that you expect.
 
+#### Effortless Infra Project
```

Comment:
This section has some inconsistent capitalization but I'm not sure what our standards are (Node Ingest Rule? node ingest rule? Do we capitalize 'node'? I have no idea.

Reply:
Thank you for catching that. I updated it to be consistent with the lower case 'node'.